src/Compilers/VisualBasic/Portable/Semantics/Conversions.vb

```diff
@@ -3472,19 +3487,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     'From a type T? to a type S?
                     conv = ClassifyPredefinedConversion(srcUnderlying, dstUnderlying, useSiteDiagnostics)
                     Debug.Assert((conv And ConversionKind.VarianceConversionAmbiguity) = 0)
+                    Debug.Assert((conv And ConversionKind.DelegateRelaxationLevelMask) = 0 OrElse (conv And ConversionKind.Tuple) <> 0)
 
                     If IsWideningConversion(conv) Then
                         'From a type T? to a type S?, where there is a widening conversion from the type T to the type S.
-                        Return ConversionKind.WideningNullable
+                        Return ConversionKind.WideningNullable Or (conv And (ConversionKind.Tuple Or ConversionKind.DelegateRelaxationLevelMask))
```

Comment:
"FromType" because we are ignoring InvolvesNarrowingFromNumericConstant, which can only happens when converting from expression

---
In reply to: [83730258](https://github.com/dotnet/roslyn/pull/14550#discussion_r83730258) [](ancestors = 83730258)